src/datanode/src/datanode.rs

```diff
@@ -68,18 +68,31 @@ pub struct StorageConfig {
     /// The precedence order is: ttl in table options > global ttl.
     #[serde(with = "humantime_serde")]
     pub global_ttl: Option<Duration>,
+    /// The working directory of database
+    pub data_home: String,
     #[serde(flatten)]
     pub store: ObjectStoreConfig,
     pub compaction: CompactionConfig,
     pub manifest: RegionManifestConfig,
     pub flush: FlushConfig,
 }
 
+impl Default for StorageConfig {
+    fn default() -> Self {
+        Self {
+            global_ttl: None,
+            data_home: DEFAULT_DATA_HOME.to_string(),
+            store: ObjectStoreConfig::default(),
+            compaction: CompactionConfig::default(),
+            manifest: RegionManifestConfig::default(),
+            flush: FlushConfig::default(),
+        }
+    }
+}
+
 #[derive(Debug, Clone, Serialize, Default, Deserialize)]
 #[serde(default)]
-pub struct FileConfig {
-    pub data_home: String,
-}
+pub struct FileConfig {}
```

Comment:
Can we remove `FileConfig` struct? or keep it as `data home` set a `running home` in `DatanodeOptions`.

Reply:
We can't remove it, because we have to know if we are using the local file storage.